mahout/src/main/resources/interpreter-settings.json

```diff
@@ -0,0 +1,58 @@
+[
+  {
```

Comment:
keeping this here in hopes of using new dynamic terp registration (unable to do so far)